https/users.go

```diff
@@ -37,14 +38,29 @@ func validateUsers(configPath string) error {
 }
 
 type userAuthRoundtrip struct {
-	tlsConfigPath string
-	handler       http.Handler
-	logger        log.Logger
+	tlsConfigPath   string
+	handler         http.Handler
+	logger          log.Logger
+	failuresCounter prometheus.Counter
+}
+
+func (u *userAuthRoundtrip) instrument(r prometheus.Registerer) {
+	u.failuresCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "prometheus_toolkit_web",
+			Name:      "http_request_basic_authentication_failures_total",
```

Comment:
Maybe match the namespace and subsystem:

```suggestion
			Namespace: "prometheus_toolkit",
			Subsystem: "https",
			Name:      "request_basic_authentication_failures_total",
```

Reply:
nice, done!